### PR TITLE
Updates user responses for gender/education

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-2354](https://openedx.atlassian.net/browse/DOC-2354)

Updates the labels for certain gender and education fields. See https://github.com/edx/edx-platform/pull/10257/files , merged in October.
### Reviewers
- [ ] Subject matter expert: @sarina 
- [x] Subject matter expert: @marcotuts 
- [x] Doc team review (sanity check): @pdesjardins or @srpearce 

FYI: @dsjen 
### Testing
- [x] Ran make html without warnings or errors
### Post-review
- [ ] Squash commits

### CODE REVIEW

**Comment by marcotuts** [comment]
:+1:

File: docs/en_us/dashboard/source/Reference.rst [hunk 1]
```diff
@@ -219,8 +219,8 @@ For information about viewing student demographic data in edX Insights, see
      * - Band
        - Student Response
      * - High school diploma or less
-       - None, Elementary/primary school, Junior secondary/junior high/middle
-         school, Secondary/high school
+       - No Formal Education, Elementary/primary school, Junior
```
Comment (srpearce) [inline docs/en_us/dashboard/source/Reference.rst:222]: Is there a reason we capitalize every word of "No Formal Education" but not other multi-word options?

**Comment by srpearce** [comment]
One nit; otherwise :+1:

File: docs/en_us/dashboard/source/Reference.rst (hunk 1 above)
Comment (lamagnifica) [inline docs/en_us/dashboard/source/Reference.rst:222]: I am folloing the capitalization used on https://courses.edx.org/register

Comment (srpearce) [inline docs/en_us/dashboard/source/Reference.rst:222]: Oh. Sigh.

**Pull request merged**